packages/stryker-mocha-runner/src/MochaTestRunner.ts

```diff
@@ -1,36 +1,53 @@
-import { getLogger } from 'log4js';
-import { EventEmitter } from 'events';
+import { getLogger, setGlobalLogLevel } from 'log4js';
 import { TestRunner, RunResult, RunStatus, RunnerOptions } from 'stryker-api/test_runner';
-import { FileDescriptor } from 'stryker-api/core';
 import LibWrapper from './LibWrapper';
 import StrykerMochaReporter from './StrykerMochaReporter';
 import MochaRunnerOptions, { mochaOptionsKey } from './MochaRunnerOptions';
+import RequireCacheRecorder from './RequireCacheRecorder';
+import { evalGlobal } from './utils';
 
-export default class MochaTestRunner extends EventEmitter implements TestRunner {
-  private files: FileDescriptor[];
+const DEFAULT_TEST_PATTERN = 'test/*.js';
```

Comment:
We should describe this in the README

Reply:
As well as how we use `this.mochaRunnerOptions.files`

Reply:
Yes! I've rewritten the readme. I've taken a page from the [configuring jest](https://facebook.github.io/jest/docs/en/configuration.html) layout, let me know what you think.